tests/playwright.config.ts

```diff
@@ -0,0 +1,91 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+	testDir: './tests',
+	/* Run tests in files in parallel */
+	fullyParallel: true,
+	/* Fail the build on CI if you accidentally left test.only in the source code. */
+	forbidOnly: !!process.env.CI,
+	/* Retry on CI only */
+	retries: process.env.CI ? 2 : 0,
+	/* Opt out of parallel tests on CI. */
+	workers: process.env.CI ? 1 : undefined,
+	/* Reporter to use. See https://playwright.dev/docs/test-reporters */
+	reporter: 'html',
+	/* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+	use: {
+		headless: true,
+		launchOptions: {
+			// slowMo: 100 /* TODO disable after recording all videos */,
+		},
+		contextOptions: {
+			recordVideo: { dir: 'videos/' } /* TODO disable in production */,
```

Comment:
we can't forget to disable this again, once we finish the tests, otherwise it will also record videos on the server and cause extra overhead
maybe add a line under the last test in the overview: disable test recording in playwright config file so we remember it